### ISSUE TITLE
[Suggestions] Better training and more info on farming

### ISSUE THREAD

**Issue by Jubetsu90** [Heishe/idleraiders-second-run#516]
As the raiders have tons of stats you are fiddling with when gearing up, it's important to know whether your setups really work or not. The current Raid training is useless, it only allows comparing total damage growth. 

What we really need is dps stats for each raider and some kind of reporting about skill procs. I need to know how many times enemy was ignited or how many times my cooldown was sped up with Time Pressure. This might be easiest with a single raider on the field at once.

Also for priests, you need to be able to test your healing somehow. And as we all know, there are different kinds of healing. Pairing up with the above suggestion, you could make priest targets friendly allies instead. You could still choose between multiple / single target to test your aoe healing and single target.

On an unrelated note, we need some information on farming levels. How much coins per enemy? How many enemies? Gold per second? How fast are they regenerating? Why not just show us the full stats on them when we click the icon on top left.

Fun game though, nice work

**Comment by askebv** [comment]
Agree with many of these things. Coins per enemy is always the same though (except bosses).

**Comment by TravisGesslein** [comment]
Regarding damage and healing: Patch 1.1 will bring a combat meter that shows damage/healing (and potentially other things) for each raider, along with information on where that damage came from. It'll look like this:

![combatmeter](https://cloud.githubusercontent.com/assets/2046189/23769983/889022a2-0511-11e7-9fbe-75116a4f70f1.png)

(BasicRanged is just the internal name of the standard archer shot, because it has no title yet [didn't need one so far])

Regarding the other stats: Agree! Will try to figure out a way to display this stuff efficiently.

**Comment by Fkscorpion** [comment]
Will that Combat Meter also be available in raids or only inside the training range?

I hope it is available in raids, so we can see them perform under "real conditions" :)

**Comment by TravisGesslein** [comment]
Yup, that will always be available. Of course, that brings up the question whether or not the training range is still needed at all. I think the answer is still yes. Raid bosses often have small things in their behavior that often make the results you get a bit different from what you get via a static training puppet. So I think the meter + training facility complement each other well.

**Comment by Jubetsu90** [comment]
Meter would be a great improvement, but training range will still be needed for testing different sets of equipment, skills etc. However i'd still point out that we need to know exactly what happens in combat, for example: who are the enemies attacking, which enemies are immolated and at which points, when did mage cast additional fireballs, when did a crit proc... etc etc etc, also same for healing stuff. We are dealing with lots of stats here and lots of moving parts so to make changes we need to be aware of what's going on
